scripts/commit-api-spec-change-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+echo "Running update changelog script"
+changeLog="$1"
+echo "$changeLog"
+node scripts/update-change-log.js "$changeLog"
+echo "Git configurations"
+git config --global user.email "team_interfaces+github@twilio.com"
+git config --global user.name "twilio-dx"
+git add -A
+if [ -n "$(git status --porcelain)" ]; then
+  branch=$(git branch --show-current)
+  echo "Current branch: $branch"
+  echo "There are changes to commit.";
+  commitMessage=''
+  if [ "$2" == 0 ] || [ "$2" == 1 ]
```

Comment:
[nit] consider setting `versionType` and using that instead of "$2". I feel it's much more readable.